src/annotations/AnnotationParser.php

```diff
@@ -13,6 +13,10 @@
 
 namespace mindplay\annotations;
 
+if (!AnnotationManager::$traitsSupported) {
+    define('mindplay\\annotations\\T_TRAIT', -2);
```

Comment:
Tried that! `defined(T_TRAIT)` is... always false. Isn't that odd?

```php
echo PHP_VERSION . ': ' . var_export(defined(T_TRAIT), true) . ' ' . T_TRAIT;
// 5.6.16: false 359
```

I use -2 instead of 369 because http://php.net/manual/en/tokens.php warns

> All tokens listed below are also defined as PHP constants. Their value is automatically generated based on PHP's underlying parser infrastructure. This means that the concrete value of a token may change between two PHP versions. For example the `T_FILE` constant is 365 in PHP 5.3, while the same value refers now to `T_TRAIT` in PHP 5.4 and the value of `T_FILE` is 369. This means that your code should never rely directly on the original `T_*` values taken from PHP version X.Y.Z, to provide some compatibility across multiple PHP versions. Instead your code should utilize custom values (using big numbers like 10000) and an appropriate strategy that will work with both PHP versions and `T_*` values.